assets/layers/rainbow_crossings/rainbow_crossings.json

```diff
@@ -34,7 +39,7 @@
         "fr": "un passage"
       },
       "tags": [
-        "highway=crossing"
+        "footway=crossing"
```

Comment:
As you changed the 'OsmTags' to include both `highway=footway` _and_ `footway=crossing`, the preset is not correct anymore. A point created with this footway will not be picked up by the theme.

(Note that this was picked up by the automated checks, causing the build to fail)